cosmos/providers/dbt/core/operators/base.py

```diff
@@ -71,8 +63,6 @@ class DbtBaseOperator(BaseOperator):
         "selector",
         "vars",
         "models",
-        "profiles_dir",
-        "profile",
```

Comment:
This looks like a braking change. Why are we removing these parameters from the `Operator` level?